stories/blizzards.stories.mdx

```diff
@@ -0,0 +1,703 @@
+name: When Winter Rages
+description: "Exploring the Power and Impact of Blizzards"
+media:
+  src: ::file ./blizzards-cover.png
+  alt: Near-blizzard conditions in Maria Stein, Ohio on December 23, 2022.
+  author:
+    name: Andrew Blackford
+pubDate: 2025-02-28
+taxonomy:
+  - name: Topics
+    values:
+      - Natural Disasters
+      - Snow
+  - name: Source
+    values:
+      - Community Contributed
+---
+
+<Block>
+  <Prose>
+
+    **Authors:** 
+    
+    Andrew Blackford<sup>[1]</sup>
+
+    <sup>[1]</sup> The University of Alabama in Huntsville (UAH)
+
+    **Disclaimer:** 
+    
+    <mark>🚧 This Data Story presents work in progress and not peer-reviewed results. 🚧</mark>
+  </Prose>
+</Block>
+
+<ScrollytellingBlock>
+  <Chapter
+    center={[-110,40]}
+    zoom={3.5}
+    datasetId='blizzard-count'
+    layerId='blizzard-count'
+    datetime='2021-12-31'
+  >
+    
+  ### Introduction
+
+  Blizzards are extreme winter weather events defined by the National Weather Service (NWS) as a combination of heavy falling or blowing snow, strong winds sustained at 35 mph or higher, and reduced visibility (less than a quarter of a mile) lasting for at least three contiguous hours. These powerful storms can bring everyday life to a standstill by disrupting travel, damaging infrastructure, and exposing people and animals to dangerous elements.  
+  
+  <Figure>
+    <Image
+      src={new URL('./blizzard-art-conditions.gif', import.meta.url).href}
+      alt='Artistic rendition of a winter storm intensifying to blizzard conditions.'
+    />
+    <Caption>
+    Artisitic rendition of a winter storm intensifying to blizzard conditions. Credit Jennifer Geary.
+    </Caption>
+  </Figure> 
+      
+  In the United States, blizzards occur most frequently in the northern Great Plains, but they have been reported in every state except Hawaii. While blizzards are traditionally associated with winter, records show they can happen any time between October and June. Advances in weather monitoring and forecasting have improved our ability to detect and report these events, which has led to an increase in reported blizzard activity over the past several decades. [On average, 13 blizzards occur annually across CONUS, but the past few decades have seen this number inflate to 19](https://www.google.com/url?q=https://static1.squarespace.com/static/58b98f7bd1758e4cc271d365/t/5c5b279c24a69469aa5a4d44/1549477788244/Colemanandschwartz.pdf&sa=D&source=docs&ust=1738269695252270&usg=AOvVaw2G74skUn4wS7sPosLCLuF8). The geographic spread of these storms has also widened, with more blizzards occurring outside their historical hotspots. Although the intensity of impact by individual storms has generally decreased due to better forecasting, their economic and societal impacts have risen, with associated losses now exceeding $1 billion per year. Understanding blizzards and preparing for their impacts is critical for communities in regions prone to extreme winter weather.
```

Comment:
can you use the direct url instead of the google referral url:
https://static1.squarespace.com/static/58b98f7bd1758e4cc271d365/t/5c5b279c24a69469aa5a4d44/1549477788244/Colemanandschwartz.pdf